mainapp/migrations/0039_auto_20180818_0909.py

```diff
@@ -0,0 +1,43 @@
+# Generated by Django 2.1 on 2018-08-18 03:39
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('mainapp', '0038_auto_20180817_2159'),
```

Comment:
You are one dependency behind. Please use 0039_auto_20180818_0328 (current leaf) and not 0038_auto_20180817_2159.